### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by ffoocn** [navidrome/navidrome#3057] [Ideas]
Is it possible to add several APIs in future versions?

1. API for managing users:
   1. Create a user through the API
   2. Delete a user through the API

**Comment by deluan** [comment]
You can already do this using the Native API. 

For the Subsonic endpoints, this won't be implemented as it is too insecure as of the current specification. I may implement it if OpenSubsonic can agree on a secure authentication/authorization schema.
